boot/bootstate20.go

```diff
@@ -375,6 +346,29 @@ func (bs20 *bootState20Base) commit() error {
 // generic methods
 //
 
+// genericSetNext implements the generic logic for setting up a snap to be tried
+// for boot and works for both kernel and base snaps (though not
+// simultaneously).
+func genericSetNext(b bootState, currentStatus string, next snap.PlaceInfo) (setStatus string, err error) {
```

Comment:
afaict this is not using currentStatus anymore

Reply:
yes I missed this one. fixed now.